backend/src/index.js

```diff
@@ -20,6 +21,7 @@ app.use(rateLimit({windowMs: 60000, max: 100}))
 app.use(express.json())
 
 app.use(authRouter)
+app.use(usersRouter)

```

Comment:
Missing `/users`

Reply:
Deleted my user-router, we are using only Francisco's router.